ISISComputingGroup/EPICS-ioc
# Ticket 4623: Attocube commission

### Description of work

See https://github.com/ISISComputingGroup/IBEX/issues/4623

### To test

* Confirm the usual motor settings are autosaved on every axis
* Confirm `WR_AMPL` is autosaved for each axis 

---

#### Code Review

    - [PV naming](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/PV-Naming).
    - [Disable records](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Disable-records)
    - [Record simulation](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Record-Simulation)
    - [Finishing touches](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/IOC-Finishing-Touches)

### Functional Tests

- IOC responds correctly in:

### Final steps
